files/zh-cn/glossary/accent/index.md

```diff
@@ -1,11 +1,13 @@
 ---
 title: 强调色
 slug: Glossary/Accent
+l10n:
+  sourceCommit: 7cec92742c6a79071626cb0fbfd0acde425e5814
```

Comment:
The sourceCommit should refer the last commit of the corresponding file: https://github.com/mdn/content/commits/main/files/en-us/glossary/accent/index.md

So, fo this one: `ada5fa5ef15eadd44b549ecf906423b4a2092f34`

Reply:
hah…? not the filehash but the commit hash?😭

Reply:
Note the name of this, "sourceCommit" which contains "commit"

Reply:
> then those i commited and merged needs an update…

Plase open PRs for them. Thanks